include/sys/spa_impl.h

```diff
@@ -226,6 +226,7 @@ struct spa {
 	metaslab_class_t *spa_dedup_class;	/* dedup allocation class */
 	uint64_t	spa_first_txg;		/* first txg after spa_open() */
 	uint64_t	spa_final_txg;		/* txg of export/destroy */
+	boolean_t	spa_exporting;	/* is the pool being exported? */
```

Comment:
Can you move this after `spa_is_initializing` which is also a `boolean_t`, this way it won't inflate the `spa_t` by a `uint64_t`.  Change to the name to `spa_is_exporting` would be nice for consistency too, what do you think about:

```
         boolean_t       spa_is_initializing;    /* true while opening pool */
+        boolean_t       spa_is_exporting;       /* true while exporting pool */
```

Reply:
Change applied